history/characters/250000_harpy.txt

```diff
@@ -10699,6 +10654,81 @@ mynta_talonscreech={
 	}
 }
 
+ripper={
+	name=Ripper
+	female=yes
+	dynasty=bloodfury
+	culture=harpy religion=yoggsaron_worship
+	martial=6 diplomacy=3 stewardship=4 intrigue=5 learning=4
+	trait=education_martial_3 trait=sadistic trait=just trait=ambitious
+	566.9.15={ birth=yes trait=creature_harpy }
+	583.1.1 = {
+		effect={
+			set_variable = { name = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_physical_trait_value }
+			set_variable = { name = wc_strength_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_physical_trait_value }
+		}
+	}
+	608.5.10={
+		death = {
+			death_reason = death_battle
+		}
+	}
+}
+
+riven={
+	name=Riven
+	female=yes
+	dynasty=bloodfury
+	culture=harpy religion=yoggsaron_worship
+	martial=6 diplomacy=3 stewardship=4 intrigue=8 learning=4
+	trait=education_martial_3 trait=content trait=greedy trait=callous
+	570.4.10={ birth=yes trait=creature_harpy }
+	583.1.1 = {
+		employer = ripper
+	}
+	608.5.14={
+		death = {
+			death_reason = death_battle
+		}
+	}
+}
+
+sister_rathtalon={
+	name=Rathtalon
+	female=yes
+	culture=harpy religion=yoggsaron_worship
+	martial=5 diplomacy=4 stewardship=4 intrigue=6 learning=6
+	trait=education_learning_3 trait=sadistic trait=greedy trait=ambitious
+	570.4.10={ birth=yes trait=creature_harpy }
+	573.1.1 = {
+		employer = 250627
+	}
+	608.5.14={
+		death = {
+			death_reason = death_battle
+		}
+	}
+}
+
+marion_wormwing={
+	name=Marion
+	female=yes
+	dynasty=wormwing
+	culture=harpy religion=kaldorei_religion
```

Comment:
Why is she Elunic?

Reply:
as in the document, wormwings lived with night elves in relative peace and Marion is their vassal.

Reply:
I see
I think she should become independent and Yogg Saroni around Cataclysm since it says they went into a frenzy

Reply:
She switches in 611.